<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="root_title">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>10</y>
     <width>671</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="all_sdept_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>学院管理</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_word">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>495</y>
     <width>841</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="all_user_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>891</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableWidget" name="sdept_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>851</width>
       <height>211</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>全选</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学院id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学院名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学院院长</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="add_sdept_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加学院</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_sdept_button">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>删除学院</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_user_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加user账户</string>
     </property>
    </widget>
    <widget class="QWidget" name="delete_sdept" native="true">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>270</y>
       <width>541</width>
       <height>151</height>
      </rect>
     </property>
     <widget class="QPushButton" name="delete_sdept_button2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>确定删除</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>选择上面的学院，进行删除</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>所有学院信息</string>
     </property>
    </widget>
    <widget class="QWidget" name="add_sdept" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>421</width>
       <height>131</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>401</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>学院id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="add_sdept_id"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>学院名称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="add_sdept_name"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>学院院长</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="add_sdept_username"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="add_sdept_button2">
         <property name="text">
          <string>确认添加</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="select_user" native="true">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>10</y>
       <width>911</width>
       <height>411</height>
      </rect>
     </property>
     <widget class="QTableWidget" name="user_table">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>841</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>全选</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>密码</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>621</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>所有用户信息(注意，添加user操作，尽量去单个的管理界面中添加）</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_user">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>删除user</string>
      </property>
     </widget>
     <widget class="QWidget" name="delete_user_widget" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>271</width>
        <height>80</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>请选择上面的用户，进行删除 </string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_user2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确认删除</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
